Templates/DefaultProject/template.json

```diff
@@ -358,6 +362,18 @@
             "file": "autoexec.cfg",
             "isTemplated": false
         },
+        {
+            "file": "package.bat",
```

Comment:
I kinda wish we had used the name export instead of package but that's just an opinion.

Reply:
its not too late

Reply:
I do like the consistency of using the term `export`, it's generally what we use in our docs and ... well the code, but it is just an opinion.

Reply:
id second the notion of using `export`. I'd recommend making it a single word, like `export.bat` and `export.sh`

Reply:
renamed to export